src/components/Nav/Nav.js

```diff
@@ -54,31 +54,31 @@ const Nav = () => {
           <div className={tab} onMouseEnter={hover} onMouseLeave={leave}>
             <div className="nav-menu">
               <Link
-                to={`/productlist/all?offset=0&limit=16`}
+                to={`/product-list/all?offset=0&limit=16`}
```

Comment:
중괄호 없이 string으로 사용해주세요
```suggestion
                to"/product-list/all?offset=0&limit=16"
```